packages/theming/theme-tokens/src/getTokens.ts

```diff
@@ -13,6 +13,8 @@ export function getAliasTokens(mode: AppearanceOptions) {
     return lightAliasTokens;
   } else if (mode === 'dark') {
     return darkAliasTokens;
+  } else if (mode === 'darkElevated') {
+    return darkAliasTokens;
```

Comment:
Optional could do mode === 'dark' && mode === 'darkElevated' on line 14 above

Reply:
`||` not `&&` I think, but otherwise yes!